tests/dialects/test_tosa.py

```diff
@@ -38,3 +38,7 @@ def test_are_tosa_broadcastable():
 def test_valid_cases():
     AddOp(operands=[tensor_i, tensor_i], result_types=[tensor_i.type]).verify_()
     AddOp(operands=[tensor_i, tensor_flat], result_types=[tensor_i.type]).verify_()
+    SubOp(operands=[tensor_i, tensor_i], result_types=[tensor_i.type]).verify_()
+    SubOp(operands=[tensor_i, tensor_flat], result_types=[tensor_i.type]).verify_()
+    MulOp(operands=[tensor_i, tensor_i], result_types=[tensor_i.type]).verify_()
+    MulOp(operands=[tensor_i, tensor_flat], result_types=[tensor_i.type]).verify_()
```

Comment:
I think we can actually delete this since we test the valid cases in the `ops.mlir` file